dev-tools/templates/.golangci.yml

```diff
@@ -12,6 +12,12 @@ issues:
   # Set to 0 to disable.
   # Default: 50
   max-issues-per-linter: 50
+  exclude-rules:
+    # Exclude package name contains '-' issue because we have at least one package with
+    # it on its name.
+    - linters:
+        - stylecheck
+      text: "ST1003:"
```

Comment:
```suggestion
      text: "ST1003:" # Poorly chosen identifier
```

Reply:
I'll mark this one as resolved because fixing the template will also fix this file.